src/environmentd/src/http.rs

```diff
@@ -78,7 +76,6 @@ pub struct TlsConfig {
 #[derive(Debug, Clone, Copy)]
 pub enum TlsMode {
     Require,
-    AssumeUser,
 }
```

Comment:
Seems like perhaps this should just become a bool?

Reply:
Are you opposed to a two variant enum?

Reply:
`TlsMode::Disabled` and `TlsMode::Enabled`? That seems preferable to a one variant enum, for sure!

Reply:
We can probably get rid of each service having its own TlsMode also, unless you know of a reason to keep that.

Reply:
There's no good place to put a shared definition, is why it is the way that it is.

Reply:
Done.